### ISSUE TITLE
Django 2.0 support

### ISSUE THREAD

**Issue by pypetey** [mbi/django-rosetta#193]
Django runserver won't start when trying to run on django 2.0rc1

```python
from .views import (home, list_languages, download_file, lang_sel, translate_text, ref_sel)
 
File "\env\Lib\site-packages\rosetta\views.py", line 4, in <module>
    from django.core.urlresolvers import reverse
ImportError: No module named 'django.core.urlresolvers'

```

**Comment by mbi** [comment]
HEAD si ready for Django 2.0, I'll release a new version to pypi as soon as 2.0 lands, or you can use HEAD if you are using 2.0rc in the meantime.

**Comment by mimischi** [comment]
@mbi Any update on this?

**Comment by mbi** [comment]
@mimischi version 0.7.14 was released 9 days ago, and supports Django 2.0
